__tests__/app.test.js

```diff
@@ -4,6 +4,7 @@ const db = require("../db/connection");
 const testData = require("../db/data/test-data/index.js");
 const seed = require("../db/seeds/seed.js");
 const allEndPoints = require("../endpoints.json");
+const { toBeSortedBy } = require("jest-sorted");
```

Comment:
This shouldn't need to be required in to use

Reply:
when i remove it, toBeSortedBy no longer works